spec/element_base_spec.rb

```diff
@@ -134,11 +140,33 @@
       )
     end
 
+    it 'skips text belonging to a parent element' do
+      expect(reference.previous).to match_normalized_html(
+        <<~HTML
+          <div data-type="note">Reference 1</div>
+        HTML
+      )
+    end
```

Comment:
maybe instead of "skips text belonging to a parent" it's more accurate to say that this "skips sibling text nodes" or "only returns element sibling"